.prettierrc

```
@@ -0,0 +1,6 @@
+{
+  "printWidth": 80,
```

Even relatively small screens can handle a little wider than 80.

```suggestion
  "printWidth": 100,
```